.setup/bin/refresh_vagrant_workers.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import pwd
+import json
+import subprocess
+from collections import OrderedDict
+
+SUBMITTY_REPOSITORY = os.environ['SUBMITTY_REPOSITORY']
+SUBMITTY_INSTALL_DIR = os.environ['SUBMITTY_INSTALL_DIR']
+DAEMON_USER = os.environ['DAEMON_USER']
+SUPERVISOR_USER = 'submitty'
+
+vagrant_workers_path = os.path.join(SUBMITTY_REPOSITORY, '.vagrant', 'workers.json')
+autograding_workers_path = os.path.join(SUBMITTY_INSTALL_DIR, 'config', 'autograding_workers.json')
+
+print("Loading existing data...")
+with open(vagrant_workers_path) as file:
+  vagrant_workers_data = json.load(file, object_pairs_hook=OrderedDict)
+
+with open(autograding_workers_path) as file:
+  autograding_workers_data = json.load(file, object_pairs_hook=OrderedDict)
+
+if 'version' in vagrant_workers_data and type(vagrant_workers_data['version']) == int:
+  provider = vagrant_workers_data['provider']
+  vagrant_workers_data = vagrant_workers_data['workers']
+else:
+  print("This script requires a worker configuration of v2 or greater. Please regenerate your configuration with 'vagrant workers generate'.")
+  exit(1)
+print("Done loading data")
+print()
+
+print("Generating SSH credentials...")
```

Comment:
add a comment explaining what steps from the (non-vagrant) sysadmin instructions these is mimicing.
https://submitty.org/sysadmin/installation/worker_installation